sdk/core/perf/src/base_test.cpp

```diff
@@ -285,7 +287,23 @@ namespace Azure { namespace Perf {
       }
       if (clientSecret.empty())
       {
-        m_testCredential = std::make_shared<Azure::Identity::DefaultAzureCredential>();
+        try
```

Comment:
This should be a chained token credential. See comment above.

Reply:
can't be , azure pipeline creds fails due to lack of envs and their content